subsys/net/lib/coap/coap_link_format.c

```diff
@@ -653,7 +653,7 @@ int coap_well_known_core_get(struct coap_resource *resource,
 			     uint8_t *data, uint16_t len)
 {
 	struct coap_option query;
-	uint8_t token[8];
+	uint8_t token[COAP_TOKEN_MAX_LEN];
```

Comment:
this change is not related with this commit, is it ?

Reply:
I've fixed a few other nits that I spotted near the commit area. Shall I move them to a separated commit?

Reply:
I think it is better